documentation/modules/adding-migration-hook.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: PROCEDURE
+[id="adding-migration-hook_{context}"]
+= Adding a migration hook to a migration plan
+
+You can a migration hook to an existing migration plan. Although you can complete part of the procedure for adding the hook to migration plan by using either the CLI or UI, one step must be done using the CLi.
+
+.Prerequisites
+
+* Migration plan
+* Migration hook image or file (discuss)
+* File containing the `Secret` for the source provider
+* {ocp} service account that has at least Write-access permission for the namespace you are working in
```

Comment:
```suggestion
* {ocp} service account that has at least *write access* permission for the namespace you are working in
```

Looking at vmware-prerequisites.adoc, you have previously used `write access`. Then go with italics or bold